docs/changes/newsfragments/7212.new

```diff
@@ -0,0 +1 @@
+Added ``export_datasets_and_create_metadata_db()`` function to export all datasets from a database to NetCDF files and create a new metadata-only database. This enables users to maintain lightweight databases for metadata while relying on NetCDF files for data analysis, significantly reducing storage requirements for large experimental datasets.
```

Comment:
can you make this mentioon of export_datasets_and_create_metadata_db a proper clickable sphinx reference?